db/migrate/20170321171136_add_uri_extra_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddUriExtraToItems < ActiveRecord::Migration[5.0]
+  def change
+    add_column :items, :uri_extra, :string
```

Comment:
I had debated over what to call that column. I like uri_fragment! Changed and pushed.